src/App.jsx

```diff
@@ -1,14 +1,32 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+
+function getUserById(userId) {
+  return usersFromServer.find(user => userId === user.id) || null;
+}
+
+function getCommentById(id) {
+  return commentsFromServer.filter(comments => id === comments.postId) || null;
```

Comment:
```suggestion
  return commentsFromServer.filter(comments => id === comments.postId);
```
`filter` always returns an array, so this condition can't be reached